remote/remote_test.go

```diff
@@ -502,36 +503,48 @@ func testImage(t *testing.T, when spec.G, it spec.S) {
 			})
 		})
 
-		when("#RetryLogic", func() {
+		when("retry logic", func() {
 			var mockServer *h.MockServer
 
-			when("Manifest API in registry returns status code 200", func() {
+			when("manifest API in registry returns status code 200", func() {
 				it.Before(func() {
 					mockServer, repoName = h.SetUpMockServer(t, "org/do-not-retry", http.StatusOK, 0)
 				})
 
-				it("Do not retry after status code 200", func() {
-					verifyRetryLogic(t, mockServer, repoName, 1)
+				it.After(func() {
+					defer mockServer.Server().Close()
+				})
+
+				it("does not retry after status code 200", func() {
+					assertExpectedRetries(t, mockServer, repoName, 1)
```

Comment:
```suggestion
					assertExpectedTries(t, mockServer, repoName, 1)
```

Reply:
(I'm sorry, I know I originally suggested `assertExpectedRetries` - but the current phrasing makes me think that we still retry once even on a successful request!)